src/utils/location.js

```diff
@@ -0,0 +1,15 @@
+const getCurrentPosition = (options = {}) =>
+  new Promise((resolve, reject) => {
+    navigator.geolocation.getCurrentPosition(resolve, reject, options);
+  });
+
+export const getLocation = async () => {
+  let position;
+  try {
+    position = await getCurrentPosition();
+  } catch (error) {
+    throw new Error(error);
+  } finally {
+    return position; // eslint-disable-line
+  }
+};
```

Comment:
there's something really weird about this file.

Reply:
done